### DISCUSSION TITLE
Save two options of "Select" type

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#2289] [Q&A] [answered]
Hi,

I have the problem that i want to save two options of my "Select" type called "tagTopic". 

The two options are meme,news which i defined in the pocketbase as allowed inputs. 

`{
  name: '',
  link: '',
  description: '',
  tagline: 'af',
  tagTopic: 'meme,news',
  user: 'db5s41oatgoeh0q'
}
`

Pocketbase only saves "meme" but ignores news. How can i save both "Select" types.

PS: Multiple is activated and max options are 2 :)

**Comment by ganigeorgiev** [comment]
Multiple values need to be submitted as array, aka.:

```js
await pb.collection('example').create({
    tagTopic: ["meme", "news"]
})
```

**Reply by MuslemRahimi** [reply]
Sorry i still have the issue because my formData has the following definition: 

`FormData {
  [Symbol(state)]: [
    { name: 'name', value: 'test' },
    { name: 'tagTopic[0]', value: 'meme' },
    { name: 'tagTopic[1]', value: 'news' },
    { name: 'tagline', value: 'test' },
    { name: 'thumbnail', value: [File] },
    { name: 'postType', value: 'image' },
    { name: 'user', value: 'db5s41oatgoeh0q' }
  ]
}
`

And it does not work on this format

**Reply by MuslemRahimi** [reply]
this is my current workaround to resolve this issue. not the best way but it works: 

`
import { serialize } from 'object-to-formdata';

let newPost = await locals.pb.collection('posts').create(serialize(formData));
			
			// add the tagTopic manually because serialize does not work on arrays
			await locals.pb.collection("posts").update(newPost.id, {
				"tagTopic": ['meme', 'news'],
			})`

**Reply by ganigeorgiev** [reply, marked as answer]
If you want to use `FormData` then you can call [`formData.append()`](https://developer.mozilla.org/en-US/docs/Web/API/FormData/append) for each entry, aka.:

```js
formData.append("tagTopic", "meme");
formData.append("tagTopic", "news");
```
